resources/js/components/DetailGroup.vue

```diff
@@ -15,7 +15,6 @@
             :resource-id="resourceId"
             :resource="resource"
             :field="item"
-            :errors="validationErrors"
```

Comment:
Why remove the validation Errors here ?

Reply:
Because it's a detail component. There is no validationErrors.